cleanlab/datalab/issue_finder.py

```diff
@@ -157,12 +153,16 @@ def find_issues(
             issue_types=issue_types,
         )
 
-        new_issue_managers = [
-            factory(datalab=self.datalab, **issue_types_copy.get(factory.issue_name, {}))
-            for factory in _IssueManagerFactory.from_list(list(issue_types_copy.keys()))
-        ]
+        new_issue_managers = []
+        for issue_type in issue_types_copy.keys():
+            if issue_type == "image_issue_types":
+                continue
+            factory = _IssueManagerFactory.from_str(issue_type)
+            new_issue_managers.append(
+                factory(datalab=self.datalab, **issue_types_copy.get(factory.issue_name, {}))
+            )
```

Comment:
I think we can revert this block, but make sure that `issue_types_copy` won't break the factory for the Datalab-specific issue managers.